com.unity.multiplayer.mlapi/Runtime/Core/RPCQueue/RPCQueueManager.cs

```diff
@@ -0,0 +1,549 @@
+using System;
+using System.Collections.Generic;
+using MLAPI.Serialization;
+using MLAPI.Serialization.Pooled;
+using MLAPI.Profiling;
+
+namespace MLAPI
+{
+    /// <summary>
+    /// RPCQueueManager (Singleton Class)
```

Comment:
haha, I guess this is no longer a singleton and also won't be a manager probably? :)

Reply:
Yes, that was something I missed.  It is added to the Jira task to be removed.